doc/release-notes/release-notes-26988.md

```diff
@@ -0,0 +1,11 @@
+New RPCs
+--------
+
+- A new test-only RPC `getaddrmaninfo` has been added, which returns count of 
+  addresses in new/tried table of all networks (or a given network). (#26988)
+
+Tools and Utilities
+--------
+
+- CLI -addrinfo would return known addresses including `isTerrrible` addresses
+  now since it has been reworked to utilise `getaddrmaninfo` RPC. (#26988)
```

Comment:
As far as I know, test-only RPCs, similar to debug-only bitcoind options, aren't usually mentioned in  release notes - the target group for release notes are regular users, while test-only options are meant for devs and may not be stable. Related test-only RPCs such as `addpeeraddress` weren't introduced in release notes either. So maybe only describe the changed behavior of cli -addrinfo and don't mention `getaddrmaninfo`?

Reply:
> hm, seems a little low-level for release notes. maybe something like

sounds much better! thank you!

>  the target group for release notes are regular users, while test-only options are meant for devs and may not be stable.

interesting, thanks! i've updated it.

Reply:
Seems kind of weird to change the CLI to use a "test-only" RPC in any case

Reply:
>  Seems kind of weird to change the CLI to use a "test-only" RPC in any case

@fanquake, @amitiuttarwar `-generate` CLI currently [uses](https://github.com/bitcoin/bitcoin/blob/467fa8943801911c233cb96d45282b1de10bfa90/src/bitcoin-cli.cpp#L1178) a [hidden RPC](https://github.com/bitcoin/bitcoin/blob/467fa8943801911c233cb96d45282b1de10bfa90/src/rpc/mining.cpp#L1055) (`generatetoaddress`) too. 
i don't think a normal user would be interested in the new/tried table breakdown of addresses. so hopefully it's ok to keep it hidden.

Reply:
> Maybe borrow from the current -addrinfo help, along the lines of CLI -addrinfo previously returned addresses known to the node after filtering for quality and recency. It now returns all of the addresses known to the node.

thanks @jonatack! this sounds much better.

> It might be interesting to have both with and without IsTerrible() filtering to maintain the current info returned and to be able to compare over time, either by returning a breakdown in the new RPC or by leaving -addrinfo unchanged to compare the two.

true! decided to keep the RPC/CLI in a separate PRs.